README.md

```diff
@@ -1,2 +1,4 @@
 # AdvLab
 Jupyter files and help files related to the Advanced Physics Lab at CSU-East Bay.
+
+Yes!
```

Comment:
this is a reasonable idea, but I would prefer that it says "Yep!" instead. I hope this does not sound too micromanaging.